app/src/main/java/dev/bluehouse/enablevolte/MainActivity.kt

```diff
@@ -46,13 +41,13 @@ class MainActivity : AppCompatActivity(), MainActivityProtocol {
     private final val TAG = "MainActivity"
     var shizukuData = ShizukuData(
         false, false,
-        false, false,
+        false, false, false,
         false,
         -1, ""
     )
     var binding: ActivityMainBinding? = null
 
-    private fun updateCarrierConfig(enableIMS: Boolean) {
+    private fun updateVolteConfig(enableVolte: Boolean) {
```

Comment:
Also here.
```suggestion
    private fun updateVolteConfig(enable VoLTE: Boolean) {
```

Reply:
<img width="613" alt="image" src="https://user-images.githubusercontent.com/2904494/217500734-20ebdb51-3117-4628-b39a-6d0d42f0d6a3.png">
Nevermind, let's just stick with `Volte` and `Vowifi`.